docs/v2-system-addon/data_events.md

```diff
@@ -31,6 +31,30 @@ where the "value" is encoded as:
   * Value 3: other custom URL(s)
 Two encoded integers for about:newtab and about:home are combined in a bitwise fashion. For instance, if both about:home and about:newtab were set to about:blank, then `value = 5 = (1 | (1 << 2))`, i.e `value = (bitfield of about:newtab) | (bitfield of about:newhome << 2)`.
 
+## Page takeover ping
+
+This is a ping indicating whether Activity Stream is currently being used on about:home and about:newtab. If either about:home or about:newtab is set to a custom URL, the category of the URL is included. It's submitted once upon Activity Stream initialization.
+
+```js
+{
+  "event": "PAGE_TAKEOVER_DATA",
+  "value": {
+    "home_pref": ["default", "about-blank", "web-extension", "custom-url"],
```

Comment:
Since "default", "about:blank", and "web-extension" are already covered by the heartbeat ping, perhaps we are only interested in the "custom-url" for about:home and about:newtab here? That said, this ping is only sent when either about:home or about:newtab was replaced by a custom URL. It will simplify the implementation a bit below, also save a ping for those users who stick with the default setting.

What do you think?